src/sentry/integrations/slack/actions/notification.py

```diff
@@ -116,6 +116,10 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
                     "link_names": 1,
                     "attachments": json.dumps(attachments),
                 }
+                self.logger.info(
+                    "rule.slack_post.attachments",
+                    extra={"organization_id": event.group.project.organization_id},
+                )
```

Comment:
is this all the info/data we need? Might be worth logging a little extra if it helps, and around the rest of the code area too, since the turnaround time for more logs is a bit long

Reply:
i'm interested mostly in if this logger fires at all

Reply:
if we can get an idea of how many orgs that would be nice too